### ISSUE TITLE
dock-cli up should show progress if containers need to be downloaded

### ISSUE THREAD

**Issue by tonypiper** [inviqa/dock-cli#38]
The UX is that nothing happens after running `dock-cli up` for several minutes and there is no indication that containers are being downloaded.

**Comment by sroze** [comment]
The `start` (previously `up`) command have the current behaviour since we changed it to background mode to be able to display the component table. When an image is not already downloaded, it will download the image before "releasing" the command, but without display anything because of the command uses a silent process runner.

A possible option could be that if after 5 seconds the containers aren't started (which means the `docker-compose up -d` command didn't release "focus") we start streaming the command outputs.

An example usage of an interactive process API could be:

``` php
$process = new InteractiveProcess('docker-compose up -d');
$process->disableOutput();
$process->ifTakesMoreThan(5000, function($process) {
    $process->enableOutput();
});

$this->interactiveProcessRunner->runProcess($process);
```

**Comment by sroze** [comment]
Closed by #39
